src/process.cr

```diff
@@ -129,19 +129,32 @@ class Process
     pid
   end
 
-  # The standard `IO` configuration of a process:
-  #
-  # * `nil`: use a pipe
-  # * `false`: no `IO` (`/dev/null`)
-  # * `true`: inherit from parent
-  # * `IO`: use the given `IO`
-  alias Stdio = Nil | Bool | IO
+  # How to redirect the standard input, output and error IO of a process.
+  enum Redirect
+    # Pipe the IO so the parent process can read (or write) to the process IO
+    # throught `#input`, `#output` or `#error`.
+    Pipe
+
+    # Discards the IO.
+    Close
+
+    # Use the IO of the parent process.
+    Inherit
+  end
+
+  PIPE    = Redirect::Pipe
```

Comment:
Yeah, they're constants that happen to be an enum value. It would be confusing to start having camelcase constants —thought there are some camelcase constants (e.g. under `Time`) which irritates me.

Hence why I believe all constants, including enum constants, should always be uppercase, while types are always camelcase.